help/destinations/catalog/personalization/algolia-user-segments.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Algolia User Segments
+description: Use this connector to activate audiences to Algolia for personalization and use across search and recommendations. You can then use the Algolia User Profile source to import the profiles into the Real-Time CDP to build rich audiences.
+exl-id: 9ee6d5cb-5d87-4a3c-9b87-99a9f7b8c8d8
+---
+
+# [!DNL Algolia User Segments] connection
+
+## Overview {#overview}
+
+>[!IMPORTANT]
+>
+>The Algolia User Segments destination connector and documentation page are created and maintained by the Algolia Integration Services team. For inquiries or update requests, contact them at [adobe-algolia-solutions@algolia.com](adobe-algolia-solutions@algolia.com).
+
+Use the [!DNL Algolia User Segments] connector to send Adobe Experience Platform audiences to Algolia to personalize search and recommendations. This requires the use of the [!DNL Algolia User Profiles] source connector to pull user profiles from Algolia to help build rich audiences.
+
+This tutorial provides steps to create an [!DNL Algolia User Segments] destination connection and dataflow using the Adobe Experience Platform user interface. This connector is used to deliver audiences to Algolia.
+
+![The destination catalog with the Algolia destination.](../../assets/catalog/personalization/algolia-user-segments/catalog.png)
```

Comment:
This screenshot could use a colored highlight over the Algolia card, so users can quickly identify it. Also, I see the card in the image is missing an actual logo. Can you take a new one with the Algolia logo updated?